### PR TITLE
Navbar Redesign with Icons

### PR DESCRIPTION
**Pull request by Quashnock**
- **Swapped indicator circles out with icons**
The icons I used from the vue materials UI library. I tested them by temporarily hardcoding the websocket and robot controller states, so you may want to fully test them with actual connections to the robot.
- **Placeholder ping indicator**
As requested in the discord meeting on 11/15 I added a placeholder ref for latency inside of useRobotLibStore and had the respective TS file export it. The ping indicator should update accordingly when the latency value is changed.
- **Small design changes to seperate indicator from tab selector**
I felt that if we use icons it may be a bit difficult to seperate indicators from clickable tabs, so I gave indicators a different background color and the clickable tabs a hover effect.

Please let me know if you have any questions or run into any issues!

### CODE REVIEW

File: src/store/useRoslib.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const SECONDS_TO_TIMESTAMP = 1000;
 export const useRoslibStore = defineStore('roslib', () => {
   const ros = new ROSLIB.Ros({ url: undefined });
   const isWebSocketConnected = ref<boolean>(false);
+  const latency = ref<Number>(0);
```
Comment (adamseth2) [inline src/store/useRoslib.ts:14]: Small thing with typescript. Primitive types string, number and boolean are lowercase while all others you capital the first letter. Change to "number"

**Review by adamseth2 — COMMENTED** [review]
Just merge from master and looks good

**Comment by uellenberg** [comment]
@Quashnock I resolved the merge conflict above. Make sure do a `git pull`, then make the changes Adam requested. Thanks!

**Comment by uellenberg** [comment]
Also, looks like CI failed. Make sure you have the formatter enabled. You can format everything using `npm run format`.

**Review by adamseth2 — APPROVED** [review]
Looks good!

**Pull request merged**